src/scripts/quests/Quests.ts

```diff
@@ -305,7 +305,7 @@ class Quests implements Saveable {
                 if (ql) {
                     ql.state(questLine.state);
                     if (questLine.state == QuestLineState.started) {
-                        ql.resumeAt(questLine.quest, questLine.initial);
+                        ql.resumeAt(questLine.quest, (questLine.initial instanceof Number ? questLine.initial : 0));
```

Comment:
I think this should be returning undefined, as if they already have progress in the statistic it's tracking, the quest will just auto complete.
```suggestion
                        ql.resumeAt(questLine.quest, (questLine.initial instanceof Number ? questLine.initial : undefined));
```